src/profile-logic/gecko-profile-versioning.js

```diff
@@ -17,7 +17,7 @@ import {
 } from './convert-markers';
 import { UniqueStringArray } from '../utils/unique-string-array';
 
-export const CURRENT_VERSION = 10; // The current version of the Gecko profile format.
+export const CURRENT_VERSION = 11; // The current version of the Gecko profile format.
```

Comment:
nit: please update the correct version in the commit message too

Reply:
done